ansible_events_ui/db/migrations/versions/202208291521_57d535d96a60_update_activation_table.py

```diff
@@ -48,11 +48,7 @@ def upgrade() -> None:
     )
     op.add_column(
         "activation",
-        sa.Column("activation_status", sa.String(), nullable=True),
-    )
-    op.add_column(
-        "activation",
-        sa.Column("activation_enabled", sa.Boolean(), nullable=False),
+        sa.Column("activation_status", sa.Boolean(), nullable=False),
```

Comment:
The migration causes data loss. It will fail if the activation table is not empty.